community-operators/postgresql/postgres-operator.v3.5.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,351 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    capabilities: Seamless Upgrades
+    categories: Database
```

Comment:
You may need to add an example CR to the [alm-examples](https://github.com/operator-framework/operator-lifecycle-manager/blob/master/Documentation/design/building-your-csv.md#crd-templates) field.

Reply:
Given the timeline it's not necessary to add a CR example at this time but it should be considered in the future.

Reply:
@jeremylinlin The alm-examples were omitted initially as this operator was created some time before the current operator pattern was in place. The operator maintains a set of user credentials and exposes an API that clients interact with to provision postgresql database instances, rather than creating the CRs directly. The thought was that having the examples in place could lead to confusion and possibly interfere with the intended workflow until we can get some changes made on the operator side to fit better into the current model, if that makes sense.